<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>castleWindow</class>
 <widget class="QMainWindow" name="castleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CASTLE</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="party_tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>551</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>551</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <property name="columnCount">
     <number>7</number>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Class/Race</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Race</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HP Max</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="castleFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>551</width>
      <height>431</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCastle">
    <property name="title">
     <string>Castle</string>
    </property>
    <addaction name="actionGilgamesh_Tavern"/>
    <addaction name="actionAdventurer_Inn"/>
    <addaction name="actionTemple_of_Cant"/>
    <addaction name="actionBoltac_Trading_Post"/>
    <addaction name="actionEdge_of_Town"/>
   </widget>
   <addaction name="menuCastle"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionGilgamesh_Tavern">
   <property name="text">
    <string>Gilgamesh's Tavern</string>
   </property>
  </action>
  <action name="actionAdventurer_Inn">
   <property name="text">
    <string>Adventurer's Inn</string>
   </property>
  </action>
  <action name="actionTemple_of_Cant">
   <property name="text">
    <string>Temple of Cant</string>
   </property>
  </action>
  <action name="actionBoltac_Trading_Post">
   <property name="text">
    <string>Boltac's Trading Post</string>
   </property>
  </action>
  <action name="actionEdge_of_Town">
   <property name="text">
    <string>Edge of Town</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
